SPECS/ruby/ruby.spec

```diff
@@ -8,20 +8,19 @@ Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://www.ruby-lang.org/en/
 Source0:        https://cache.ruby-lang.org/pub/ruby/2.6/%{name}-%{version}.tar.xz
+#Source0:        %{name}-%{version}.tar.gz
```

Comment:
nit - remove 
```suggestion
```

Reply:
Will remove the SOURCES folder and upload the sources file to the server before checking in the PR.